e2e/fixtures/events/createClaimSpec.js

```diff
@@ -0,0 +1,393 @@
+          respondent2OrgRegistered: 'Yes',
+          respondent2OrganisationPolicy: {
+            OrgPolicyReference: 'Defendant policy reference 2',
+            OrgPolicyCaseAssignedRole: '[RESPONDENTSOLICITORTWO]',
+            Organisation:
+
+              {OrganisationID: 'H2156A0'}
+            ,
+          },
+        },
+        SecondDefendantSolicitorServiceAddress: {
+          respondentSolicitor2ServiceAddress: buildAddress('service')
+        },
+        SecondDefendantSolicitorReference: {
+          respondentSolicitor2Reference: 'sol2reference'
+        },
+        SecondDefendantSolicitorEmail: {
+          respondentSolicitor2EmailAddress: 'civilunspecified@gmail.com'
+        }
+      };
+    }
+    case 'TWO_V_ONE': {
+      return {
+        ...claimData,
+        AddAnotherClaimant: {
+          addApplicant2: 'Yes'
+        }
+      };
+    }
+    case 'ONE_V_ONE':
+    default: {
+      return claimData;
+    }
+  }
+};
+
+const hasRespondent2 = (mpScenario) => {
+  return mpScenario === 'ONE_V_TWO_ONE_LEGAL_REP'
+      || mpScenario ===  'ONE_V_TWO_TWO_LEGAL_REP';
+};
+
+module.exports = {
+  createClaim: (mpScenario = 'ONE_V_ONE') => {
+    return {
+      midEventData: {
+        ClaimValue: {
+          applicantSolicitor1PbaAccounts: {
+            list_items: [
+              validPba,
+              invalidPba
+            ],
+            value: selectedPba
+          },
+          claimValue: {
+            statementOfValueInPennies: '3000000'
+          },
+          claimFee: {
+            calculatedAmountInPence: '150000',
+            code: 'FEE0209',
+            version: '3'
+          },
+        },
+
+
+        PbaNumber: {
+          applicantSolicitor1PbaAccounts: {
+            list_items: [
+              validPba,
+              invalidPba
+            ],
+            value: selectedPba
+          }
+        },
+
+        StatementOfTruth: {
+          applicantSolicitor1ClaimStatementOfTruth: {}
+        },
+      },
+      valid: {
+        ...createClaimData('Yes', true, mpScenario),
+      },
+      invalid: {
+        Upload: {
+          servedDocumentFiles: {
+            particularsOfClaimDocument: [
+              {
+                id: docUuid,
+                value: {
+                  document_url: '${TEST_DOCUMENT_URL}',
+                  document_binary_url: '${TEST_DOCUMENT_BINARY_URL}',
+                  document_filename: '${TEST_DOCUMENT_FILENAME}'
+                }
+              },
+              {
+                id: docUuid,
+                value: {
+                  document_url: '${TEST_DOCUMENT_URL}',
+                  document_binary_url: '${TEST_DOCUMENT_BINARY_URL}',
+                  document_filename: '${TEST_DOCUMENT_FILENAME}'
+                }
+              }
+            ]
+          }
+        },
+        Court: {
+          courtLocation: {
+            applicantPreferredCourt: ['3a3', '21', '3333']
+          }
+        }
+      }
+    };
+  },
+
```

Comment:
Commented out code should be removed or, at the very least, explain why it still has to be here

Reply:
removed redundant comments